multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+/*.yaml
+*~
```

Comment:
I guess email is better than trying to read inline. Saw your comment about *.yaml. I'd rather not have a wildcard ignores for what a person *might* do.